test/extended/setup.sh

```diff
@@ -71,7 +75,9 @@ function os::test::extended::setup {
     # If the current system has the XFS volume dir mount point we configure
     # in the test images, assume to use it which will allow the local storage
     # quota tests to pass.
-    if [ -d "/mnt/openshift-xfs-vol-dir" ]; then
+    LOCAL_STORAGE_QUOTA=""
+    if [[ -d "/mnt/openshift-xfs-vol-dir" && $(lvs | grep -q "xfs") ]]; then
```

Comment:
I guess we `os::util::environment::use_sudo` in these scripts anyway, so it should be fine to use `sudo` here